deployment/deployment.json

```diff
@@ -5151,6 +5151,36 @@
       }
     }
   },
+  "baseswap": {
+    "schema": "dex-amm",
+    "base": "uniswap-forks",
+    "protocol": "baseswap",
+    "project": "ubeswap",
+    "deployments": {
+      "baseswap-base": {
+        "network": "base",
+        "status": "prod",
+        "versions": {
+          "schema": "1.3.2",
+          "subgraph": "1.0.0",
+          "methodology": "1.0.0"
+        },
+        "files": {
+          "template": "uniswap.v2.template.yaml"
```

Comment:
change template file name to `baseswap.template.yaml`

Reply:
changed